packages/connection/src/Manager.php

```diff
@@ -1334,6 +1336,11 @@ protected function verify_secrets( $action, $secret_1, $user_id ) {
 			);
 		}
 
+		// Something went wrong during the checks, returning the error.
+		if ( null !== $error ) {
```

Comment:
This check doesn't have to be so narrow, you could just use `if ( ! empty( $error ) ) ...`

Reply:
Change made in 1f7a673.